### ISSUE TITLE
failed to download

### ISSUE THREAD

**Issue by Kuchitama** [mackerelio/ansible-mackerel-agent#5]
I failed to install this role with below message. I'm using ansible version 1.9.2.

```
$ ansible-galaxy install mackerel.mackerel-agent 
- downloading role 'mackerel-agent', owned by mackerel
- downloading role from https://github.com/mackerelio/ansible-mackerel-agent/archive/v0.1.0.tar.gz
- error: failed to download the file.
- mackerel.mackerel-agent was NOT installed successfully.
- you can use --ignore-errors to skip failed roles.
```

Maybe you should add a tag `v0.1.0` or other.
ansible-galaxy will download from github repository's archive.

see below.
https://github.com/ansible/ansible/blob/stable-1.9/bin/ansible-galaxy#L483

**Comment by yuuki1** [comment]
That is just as you point out.
We have released v0.3.0, then now you can install the latest version like below.

```
$ ansible-galaxy install mackerel.mackerel-agent
- downloading role 'mackerel-agent', owned by mackerel
- downloading role from https://github.com/mackerelio/ansible-mackerel-agent/archive/v0.3.0.tar.gz
- extracting mackerel.mackerel-agent to /usr/local/etc/ansible/roles/mackerel.mackerel-agent
- mackerel.mackerel-agent was installed successfully
```

Thank you for bringing it to my attention.

**Comment by Kuchitama** [comment]
I could installed latest version. Thanks a lot :+1:
